### ISSUE TITLE
Site links are broken

### ISSUE THREAD

**Issue by brianray** [data-describe/data-describe#452]
http://data-describe.ai/docs/master/data_describe/index.html#module-data_describe

Returns a 404

**Comment by haishiro** [comment]
https://github.com/data-describe/data-describe/runs/1703345085?check_suite_focus=true

```
Exception occurred:
  File "/usr/share/miniconda3/envs/doc-env/lib/python3.7/site-packages/pkg_resources/__init__.py", line 775, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (parso 0.7.0 (/usr/share/miniconda3/envs/doc-env/lib/python3.7/site-packages), Requirement.parse('parso<0.9.0,>=0.8.0'), {'jedi'})
The full traceback has been saved in /tmp/sphinx-err-5c1hd7v0.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/doc-env/bin/sphinx-multiversion", line 8, in <module>
    sys.exit(main())
  File "/usr/share/miniconda3/envs/doc-env/lib/python3.7/site-packages/sphinx_multiversion/main.py", line 338, in main
    subprocess.check_call(cmd, cwd=current_cwd)
  File "/usr/share/miniconda3/envs/doc-env/lib/python3.7/subprocess.py", line 363, in check_call
    raise CalledProcessError(retcode, cmd)
```

https://github.com/executablebooks/jupyter-book/issues/1151

**Comment by haishiro** [comment]
@truongc2 could you look at `etc/doc-environment.yml` to determine pinned versions that won't give this error?

**Comment by haishiro** [comment]
Fixed by #453
